app/src/main/res/values/strings.xml

```diff
@@ -203,9 +202,15 @@
     <string name="kpop">kpop</string>
     <string name="top_tracks">Top Tracks</string>
 
-    <string name="test_profile_image">bit.ly/3IUnyAF</string>
     <string name="PartyRoom_joinedPeople">Joined People:</string>
+
+    <!-- Profile -->
+    <string name="verify_change_profile_pic">Do you want to change your profile picture?</string>
     <string name="logout">logout</string>
     <string name="scores">Scores</string>
-    <string name="friends">Friends</string>
+    <string name="following">Following</string>
+    <string name="test_profile_image">bit.ly/3IUnyAF</string>
+    <string name="profile_username">Profile Username</string>
```

Comment:
According to the [wiki](https://github.com/MaximeZmt/SDP_2022-Vibester/wiki) We now use the following convention for all new Strings: nameOfActivity_purposeOfString
Which means you should have 
- profile_verifyChangeProfilePic
- profile_logout
- profile_scores
- ...

In addition you have to put them between already cleaned string and the old string. Meaning around line 42 of string.xml